site/docs/components/pill/index.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: Pill
+data:
+  description: Pill is a small visual element that contains versatile content. You can use Pill to label, tag or categorize content. With Pill, users can trigger actions, make selections or filter results. Actions that may change depending on the context, and are not fixed, are better represented by Pill than Button.
+
+  # Fill in the info from the content template's "Metadata" table below.
+  # To omit optional items, comment them out with #
+  sourceCodeUrl: "https://github.com/jpmorganchase/salt-ds/blob/pill-next/packages/core/src/pill/"
+  package:
+    name: "@salt-ds/core"
+  alsoKnownAs: ["Tag", "Chip", "Badge"]
+  relatedComponents: [
```

Comment:
Should "Contains" display nothing or the text should not be there as well?

Reply:
The heading should disappear. It's a bug in the component page layout. I'm working on a separate PR to fix that and a few other minor issues I've found on the component page layout.